test/unit/omp-target/test-nested-reduce.cpp

```diff
@@ -27,15 +27,17 @@ static RAJA::Index_type const area = xExtent * yExtent;
 
 TEST(NestedReduceTargetOMP,outer)
 {
+  RAJA::Index_type  l_begin = begin;
```

Comment:
Has this bug been reported to IBM?

Reply:
Not yet!

Reply:
I still think this is silly but we have to make it work in the meantime :)